pkg/cmd/openshift-apiserver/openshiftapiserver/openshift_apiserver.go

```diff
@@ -509,6 +511,28 @@ func (c *completedConfig) withUserAPIServer(delegateAPIServer genericapiserver.D
 	return server.GenericAPIServer, &legacyStorageVersionMutator{version: userapiv1.SchemeGroupVersion, storage: storage}, nil
 }
 
+func (c *completedConfig) withOpenAPIAggregator(delegatedAPIServer *genericapiserver.GenericAPIServer) (*genericapiserver.GenericAPIServer, error) {
+	delegatedAPIServer.RemoveOpenAPIData()
```

Comment:
add a comment why this is needed and why this is sane. Why don't we lose any important openapi specs this way from the passed API server?

Reply:
> Why don't we lose any important openapi specs this way from the passed API server?

As far as I've been able to tell, we instantiate a "default" openapi config here: https://github.com/openshift/origin/blob/master/pkg/cmd/openshift-apiserver/openshiftapiserver/config.go#L107